package.json

```diff
@@ -195,7 +201,18 @@
   },
   "targets": {
     "web": {
-      "context": "browser"
+      "context": "browser",
+      "distDir": "./build"
```

Comment:
ah, I missed this in https://github.com/oasisprotocol/oasis-wallet-web/pull/746: `build-without-minify.js` isn't used anymore and we didn't migrate.

We wanted better stack traces in bug reports: https://github.com/oasisprotocol/oasis-wallet-web/pull/498

Reply:
would like to keep optimisation for prod env, but if that's must have there is a `--no-optimize` flag. I will create PR to target master branch.